Tests/ML/datasets/test_dataset.py

```diff
@@ -419,6 +421,51 @@ def test_get_all_metadata(default_config: ModelConfigBase) -> None:
     assert PatientMetadata.from_dataframe(df, '2') == PatientMetadata(patient_id='2', institution="2")
 
 
+def test_converts_channels_to_file_paths(default_config: ModelConfigBase) -> None:
+    """
+    Test unit for missing channels and missing files.
+    """
+    # Sets DummyModel config and container
+    container = InnerEyeContainer(default_config)
+
+    # 1 Should not return any errors given that no channels or files are missing
+    assert container.setup() is None
```

Comment:
```suggestion
    container.setup()
```

Reply:
Removed assert statement. Changed to:
container.setup()